src/minecraftServers/Models/Server.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace MinecraftServers.Models;
 
+[IdCountAtt]
```

Comment:
It's a wrong way to do auto-increment. Try to change `Id` type from `uint` to `int`